<!DOCTYPE html>
        <html lang="en">
        <head>
            <meta charset="UTF-8" />
            <title>Analysis of the theme/rheme coreferences in text</title>
            <style>
                span {white-space:nowrap}
            </style>
        </head>

        <body>

            <h1>Texto</h1>

        			
			 Breast cancer exhibits familial aggregation, consistent with variation in genetic susceptibility to the disease.
		 			
			 Known susceptibility genes account for less than 25_% of the familial risk of breast cancer, and the residual genetic variance is likely to be due to variants conferring more moderate risks.
		 			
			 To identify further susceptibility alleles, we conducted a two-stage genome-wide association study in 4,398 breast cancer cases and 4,316 controls, followed by a third stage in which 30 single nucleotide polymorphisms ( SNPs ) were tested for confirmation in 21,860 cases and 22,578 controls from S22 studies.
		 			
			 We used 227,876 SNPs that were estimated to correlate with 77_% of known common SNPs in Europeans at r2  0.5.
		 			
			 SNPs in five novel independent loci exhibited strong and consistent evidence of association with breast cancer ( P  10 ( -7 ) ).
		 			
			 Four of these contain plausible causative genes ( FGFR2, TNRC9, MAP3K1 and LSP1 ).
		 			
			 At the second stage, 1,792 SNPs were significant at the P  0.05 level compared with an estimated 1,343 that would be expected by_chance, indicating that many additional common susceptibility alleles may be identifiable by this approach.
		 			
			 Breast cancer is about twice as common in the first-degree relatives of women with the disease as in the general population, consistent with variation in genetic susceptibility to the disease1.
		 			
			 In the 1990s, two major susceptibility genes for breast cancer, BRCA1 and BRCA2, were identified.
		 			
			 Inherited mutations in these genes lead to a high risk of breast and other cancers.
		 			
			 However, the majority of multiple case breast cancer families do not segregate mutations in these genes.
		 			
			 Subsequent genetic linkage studies have failed to identify further major breast cancer genes.
		 			
			 These observations have led to the proposal that breast cancer susceptibility is largely ‘ polygenic ’: that is, susceptibility is conferred by a large number of loci, each with a small effect on breast cancer risk.
		 			
			 This model is consistent with the observed patterns of familial aggregation of breast cancer.
		 			
			 However, progress in identifying the relevant loci has been slow.
		 			
			 As linkage studies lack power to detect alleles with moderate effects on risk, large case-control association studies are required.
		 			
			 Such studies have identified variants in the DNA repair genes CHEK2 ​, ATM ​, BRIP1 and PALB2that confer an approximately twofold risk of breast cancer, but these variants are rare in the population -.
		 			
			 A recent study has shown that a common coding variant in CASP8 is associated with a moderate reduction in breast cancer risk.
		 			
			 After accounting for all the known breast cancer loci, more than 75_% of the familial risk of the disease remains unexplained.
		 			
			 Recent technological advances have provided platforms that allow hundreds of thousands of SNPs to be analysed in association studies, thus providing a basis for identifying moderate risk alleles without prior knowledge of position or function.
		 			
			 It has been estimated that there are 7_million common SNPs in the human genome ( with minor allele frequency, m. a. f.,  5_% ).
		 			
			 However, because recombination tends to occur at distinct ‘ hot-spots ’, neighbouring polymorphisms are often strongly correlated ( in ‘ linkage disequilibrium ’, LD ) with each other.
		 			
			 The majority of common genetic variants can therefore be evaluated for association using a few hundred thousand SNPs as tags for all the other variants.
		 			
			 We aimed to identify further breast cancer susceptibility loci in a three-stage association study.
		 			
			 In the first stage, we used a panel of 266,722 SNPs, selected to tag known common variants across the entire genome18.
		 			
			 These SNPs were genotyped in 408 breast cancer cases and 400 controls from the UK ; data were analysed for 390 cases and 364 controls genotyped for ≥ 80_% of the SNPs.
		 			
			 The cases were selected to have a strong family history of breast cancer, equivalent to at_least two affected female first-degree relatives, because such cases are more likely to carry susceptibility alleles.
		 			
			 Initally, we analysed 227,876 SNPs ( 85_% ) with genotypes on at_least 80_% of the subjects.
		 			
			 We estimate that these SNPs are correlated with 58_% of common SNPs in the HapMap_CEPH / CEU ( Utah residents with ancestry from northern and western Europe ) samples at r2  0.8, and 77_% at r2  0.5 ( mean r2=0.75 ; see Supplementary_Fig. 1 ) ( http://www.hapmap.org/ ).
		 			
			 As expected, coverage was strongly related to m. a. f.: 70_% of SNPs with m. a. f.  10_% were tagged at r2  0.8, compared with 23_% of SNPs with m. a. f. 5 – 10_%.
		 			
			 The main analyses were restricted to 205,586 SNPs that had a call rate of 90_% and whose genotype distributions did not differ from Hardy – Weinberg equilibrium in controls ( at P  10 − 5 ).
		 			
			 For the second stage we selected 12,711 SNPs, approximately 5_% of those typed in stage 1, on the basis of the significance of the difference in genotype frequency between cases and controls.
		 			
			 These SNPs were then genotyped in a further 3,990 invasive breast cancer cases and 3,916 controls from the SEARCH study, using a custom-designed oligonucleotide array.
		 			
			 In the main analyses, we considered 10,405 SNPs with call rate of  95_% that did not deviate from Hardy – Weinberg equilibrium in controls.
		 			
			 Comparison of the observed and expected distribution of test statistics showed some evidence for an inflation of the test statistics in both stage 1 ( inflation factor λ=1.03, 95_% confidence interval ( CI ) 1.02 – 1.04 ) and stage 2 ( λ=1.06, 95_% CI 1.04 – 1.12 ), based on the 90_% least significant SNPs ( Fig. 1 ).
		 			
			 Possible explanations for this inflation include population stratification, cryptic relatedness among subjects, and differential genotype calling between cases and controls.
		 			
			 There was evidence for an excess of low call rate SNPs among the most significant SNPs ( P  0.01 ) in stage 1, but not in stage 2, suggesting that some of this effect is a genotyping artefact ( Supplementary_Table 1 ).
		 			
			 However, the inflation was still present among SNPs with call rate  99_% in both cases and controls, possibly reflecting population sub-structure.
		 			
			 We computed 1 degree of freedom ( d. f. ) association tests for each SNP, combining stages 1 and 2.
		 			
			 After adjustment for this inflation by the genomic control method, we observed more associations than would have been expected by_chance at P  0.05 ( Table 1 ).
		 			
			 One SNP ( dbSNP rs2981582 ) was significant at the P  10 − 7 level that has been proposed as appropriate for genome-wide studies.
		 

            <h1>Conceptos</h1><p style="color:#708090;">Breast cancer (c_0)</p><p style="color:#D2691E;">the residual genetic variance (c_1)</p><p style="color:#556B2F;">we (c_2)</p><p style="color:#FF3300;">SNPs in five novel independent loci (c_3)</p><p style="color:#000080;">1,292 SNPs (c_4)</p><p style="color:#2F4F4F;">large case-control association studies (c_5)</p><p style="color:#4B0082;">Recent technological advances (c_6)</p><p style="color:#8B4513;">recombination (c_7)</p><p style="color:#6A5ACD;">coverage (c_8)</p><p style="color:#663399;">The main analyses (c_9)</p><p style="color:#BDB76B;">Comparison of the observed and expected distribution of test statistics (c_10)</p><p style="color:#FFD700;">Possible explanations for this inflation (c_11)</p><p style="color:#00FF7F;">exhibits (c_12)</p><p style="color:#D2B48C;">variants (c_13)</p><p style="color:#DEB887;">more moderate risks (c_14)</p><p style="color:#32CD32;">4,316 controls (c_15)</p><p style="color:#FF436E;">a third stage (c_16)</p><p style="color:#0000CD;">confirmation (c_17)</p><p style="color:#008000;">Europeans (c_18)</p><p style="color:#AF83BE;">association (c_19)</p><p style="color:#6B8E23;">( FGFR2 (c_20)</p><p style="color:#90EE90;">the P 0.05 level (c_21)</p><p style="color:#7B68EE;">the first-degree relatives (c_22)</p><p style="color:#43825A;">the general population (c_23)</p><p style="color:#5F9EA0;">the disease1 (c_24)</p><p style="color:#826E43;">susceptibility (c_25)</p><p style="color:#6495ED;">a large number (c_26)</p><p style="color:#00CED1;">a panel (c_27)</p><p style="color:#808080;">the UK (c_28)</p><p style="color:#00008B;">Supplementary_Fig (c_29)</p><p style="color:#438281;">a call rate (c_30)</p><p style="color:#4169E1;">95_% (c_31)</p><p style="color:#FF1493;">population stratification (c_32)</p>

            <h1>Oraciones</h1><p> [ c_0 / c_12,c_1 ] <span style="white-space:nowrap;color:#708090;">Breast cancer </span> ////<br/><nobr><span style="white-space:nowrap;color:#00FF7F;">[exhibits]c_12 </span><span style="white-space:nowrap;color:#D2691E;">[familial aggregation]c_1 </span>, . </nobr><p> [ c_1 / c_13,c_14 ] <span style="white-space:nowrap;color:#D2691E;">the residual genetic variance </span> ////<br/><nobr>is likely to be due to <span style="white-space:nowrap;color:#D2B48C;">[variants]c_13 </span>conferring <span style="white-space:nowrap;color:#DEB887;">[more moderate risks]c_14 </span></nobr><p> [ c_2 / c_5,c_0,c_15,c_16,c_1,c_3,c_17,c_2 ] <span style="white-space:nowrap;color:#556B2F;">we </span> ////<br/><nobr>conducted <span style="white-space:nowrap;color:#2F4F4F;">[a two-stage genome-wide association study]c_5 </span>in <span style="white-space:nowrap;color:#708090;">[4,398 breast cancer cases]c_0 </span>and <span style="white-space:nowrap;color:#32CD32;">[4,316 controls]c_15 </span>, followed by <span style="white-space:nowrap;color:#FF436E;">[a third stage]c_16 </span>in which <span style="white-space:nowrap;color:#D2691E;">[30 single nucleotide polymorphisms]c_1 </span>( <span style="white-space:nowrap;color:#FF3300;">[SNPs]c_3 </span>) were tested for <span style="white-space:nowrap;color:#0000CD;">[confirmation]c_17 </span>in <span style="white-space:nowrap;color:#556B2F;">[21,860 cases]c_2 </span>and 22,578 controls from S22 studies . </nobr><p> [ c_2 / c_4,c_18,c_2 ] <span style="white-space:nowrap;color:#556B2F;">We </span> ////<br/><nobr>used <span style="white-space:nowrap;color:#000080;">[227,876 SNPs]c_4 </span>that were estimated to correlate with 77_% of known common SNPs in <span style="white-space:nowrap;color:#008000;">[Europeans]c_18 </span>at <span style="white-space:nowrap;color:#556B2F;">[r2]c_2 </span>0.5 . </nobr><p> [ c_3 / c_17,c_19,c_0,c_2 ] <span style="white-space:nowrap;color:#FF3300;">SNPs in five novel independent loci </span> ////<br/><nobr>exhibited strong and consistent <span style="white-space:nowrap;color:#0000CD;">[evidence]c_17 </span>of <span style="white-space:nowrap;color:#AF83BE;">[association]c_19 </span>with <span style="white-space:nowrap;color:#708090;">[breast cancer]c_0 </span>( <span style="white-space:nowrap;color:#556B2F;">[P]c_2 </span>10 ( -7 ) ) . </nobr><p> [ c_2 / c_1,c_20 ] <span style="white-space:nowrap;color:#556B2F;">these </span> ////<br/><nobr>contain <span style="white-space:nowrap;color:#D2691E;">[plausible causative genes]c_1 </span><span style="white-space:nowrap;color:#6B8E23;">[( FGFR2]c_20 </span>, TNRC9 , MAP3K1 and LSP1 ) </nobr><p> [ c_4 / c_21 ] <span style="white-space:nowrap;color:#000080;">1,292 SNPs </span> ////<br/><nobr>were significant at <span style="white-space:nowrap;color:#90EE90;">[the P 0.05 level]c_21 </span>, . </nobr><p> [ c_0 / c_22,c_2,c_23,c_13,c_1,c_24 ] <span style="white-space:nowrap;color:#708090;">Breast cancer </span> ////<br/><nobr>is about twice as common in <span style="white-space:nowrap;color:#7B68EE;">[the first-degree relatives]c_22 </span>of <span style="white-space:nowrap;color:#556B2F;">[women]c_2 </span>with the disease as in <span style="white-space:nowrap;color:#43825A;">[the general population]c_23 </span>, consistent with <span style="white-space:nowrap;color:#D2B48C;">[variation]c_13 </span>in <span style="white-space:nowrap;color:#D2691E;">[genetic susceptibility]c_1 </span>to <span style="white-space:nowrap;color:#5F9EA0;">[the disease1]c_24 </span>. </nobr><p> [ c_0 /  ] <span style="white-space:nowrap;color:#708090;">two major susceptibility genes for breast cancer , BRCA1 and BRCA2 , </span> ////<br/><nobr>were identified . </nobr><p> [ c_1 /  ] <span style="white-space:nowrap;color:#D2691E;">Inherited mutations in these genes </span> ////<br/><nobr>lead . </nobr><p> [ c_0 / c_13,c_1 ] <span style="white-space:nowrap;color:#708090;">the majority of multiple case breast cancer families </span> ////<br/><nobr>do segregate <span style="white-space:nowrap;color:#D2B48C;">[mutations]c_13 </span>in <span style="white-space:nowrap;color:#D2691E;">[these genes]c_1 </span>. </nobr><p> [ c_1 / c_0 ] <span style="white-space:nowrap;color:#D2691E;">Subsequent genetic linkage studies </span> ////<br/><nobr>have failed to identify further major <span style="white-space:nowrap;color:#708090;">[breast cancer]c_0 </span>genes . </nobr><p> [ c_2 / c_26,c_19 ] <span style="white-space:nowrap;color:#556B2F;">These observations </span> ////<br/><nobr>have led to the proposal that breast cancer susceptibility is largely ‘ polygenic ’ : that is , susceptibility is conferred by <span style="white-space:nowrap;color:#6495ED;">[a large number]c_26 </span>of <span style="white-space:nowrap;color:#AF83BE;">[loci]c_19 </span>, each with a small effect on breast cancer risk . </nobr><p> [ c_2 / c_2,c_1,c_0 ] <span style="white-space:nowrap;color:#556B2F;">This model </span> ////<br/><nobr>is consistent with <span style="white-space:nowrap;color:#556B2F;">[the observed patterns]c_2 </span>of <span style="white-space:nowrap;color:#D2691E;">[familial aggregation]c_1 </span>of <span style="white-space:nowrap;color:#708090;">[breast cancer]c_0 </span>. </nobr><p> [ c_3 /  ] <span style="white-space:nowrap;color:#FF3300;">progress in identifying the relevant loci </span> ////<br/><nobr>has been slow . </nobr><p> [ c_5 /  ] <span style="white-space:nowrap;color:#2F4F4F;">large case-control association studies </span> ////<br/><nobr>are required . </nobr><p> [ c_2 / c_1,c_14,c_0,c_23 ] <span style="white-space:nowrap;color:#556B2F;">Such studies </span> ////<br/><nobr>have identified variants in <span style="white-space:nowrap;color:#D2691E;">[the DNA repair genes]c_1 </span>CHEK2 ​ , ATM ​ , BRIP1 and PALB2that confer <span style="white-space:nowrap;color:#DEB887;">[an approximately twofold risk]c_14 </span>of <span style="white-space:nowrap;color:#708090;">[breast cancer]c_0 </span>, but these variants are rare in <span style="white-space:nowrap;color:#43825A;">[the population]c_23 </span>- . </nobr><p> [ c_5 / c_1,c_14,c_0 ] <span style="white-space:nowrap;color:#2F4F4F;">A recent study </span> ////<br/><nobr>has shown that <span style="white-space:nowrap;color:#D2691E;">[a common coding variant]c_1 </span>in CASP8 is associated with <span style="white-space:nowrap;color:#DEB887;">[a moderate reduction]c_14 </span>in <span style="white-space:nowrap;color:#708090;">[breast cancer]c_0 </span>risk . </nobr><p> [ c_0 /  ] <span style="white-space:nowrap;color:#708090;">more than 75_% of the familial risk of the disease </span> ////<br/><nobr>remains unexplained . </nobr><p> [ c_6 / c_26,c_3,c_19,c_14,c_12 ] <span style="white-space:nowrap;color:#4B0082;">Recent technological advances </span> ////<br/><nobr>have provided platforms that allow <span style="white-space:nowrap;color:#6495ED;">[hundreds of thousands]c_26 </span>of <span style="white-space:nowrap;color:#FF3300;">[SNPs]c_3 </span>to be analysed in <span style="white-space:nowrap;color:#AF83BE;">[association]c_19 </span>studies , thus providing a basis for identifying <span style="white-space:nowrap;color:#DEB887;">[moderate risk alleles]c_14 </span>without prior knowledge of <span style="white-space:nowrap;color:#00FF7F;">[position]c_12 </span>or function . </nobr><p> [ c_2 / c_4,c_1,c_14,c_13,c_19,c_20 ] <span style="white-space:nowrap;color:#556B2F;">It </span> ////<br/><nobr>has been estimated that there are <span style="white-space:nowrap;color:#000080;">[7_million common SNPs]c_4 </span>in <span style="white-space:nowrap;color:#D2691E;">[the human genome]c_1 </span>( with <span style="white-space:nowrap;color:#DEB887;">[minor allele frequency]c_14 </span>, <span style="white-space:nowrap;color:#D2B48C;">[m]c_13 </span>. <span style="white-space:nowrap;color:#AF83BE;">[a]c_19 </span>. <span style="white-space:nowrap;color:#6B8E23;">[f]c_20 </span>. , 5_% ) . </nobr><p> [ c_7 /  ] <span style="white-space:nowrap;color:#8B4513;">recombination </span> ////<br/><nobr>tends to occur at distinct ‘ hot-spots ’ </nobr><p> [ c_1 / c_19,c_3,c_1 ] <span style="white-space:nowrap;color:#D2691E;">The majority of common genetic variants </span> ////<br/><nobr>can be evaluated for <span style="white-space:nowrap;color:#AF83BE;">[association]c_19 </span>using a few hundred thousand <span style="white-space:nowrap;color:#FF3300;">[SNPs]c_3 </span>as tags for <span style="white-space:nowrap;color:#D2691E;">[all the other variants]c_1 </span>. </nobr><p> [  /  ] <span style="white-space:nowrap;color:#black;"></span> ////<br/><nobr></nobr><p> [ c_2 / c_19,c_4,c_13,c_1 ] <span style="white-space:nowrap;color:#556B2F;">we </span> ////<br/><nobr>used <span style="white-space:nowrap;color:#AF83BE;">[a]c_19 </span>panel of <span style="white-space:nowrap;color:#000080;">[266,722 SNPs]c_4 </span>, selected to tag known common <span style="white-space:nowrap;color:#D2B48C;">[variants]c_13 </span>across <span style="white-space:nowrap;color:#D2691E;">[the entire genome18]c_1 </span>. </nobr><p> [ c_3 / c_0,c_15,c_28,c_3 ] <span style="white-space:nowrap;color:#FF3300;">These SNPs </span> ////<br/><nobr>were genotyped in 408 <span style="white-space:nowrap;color:#708090;">[breast cancer]c_0 </span>cases and <span style="white-space:nowrap;color:#32CD32;">[400 controls]c_15 </span>from <span style="white-space:nowrap;color:#808080;">[the UK]c_28 </span>; data were analysed for 390 cases and 364 controls genotyped for ≥ 80_% of the <span style="white-space:nowrap;color:#FF3300;">[SNPs]c_3 </span>. </nobr><p> [ c_2 / c_0,c_22,c_13,c_25 ] <span style="white-space:nowrap;color:#556B2F;">The cases </span> ////<br/><nobr>were selected to have a strong family history of <span style="white-space:nowrap;color:#708090;">[breast cancer]c_0 </span>, equivalent to <span style="white-space:nowrap;color:#7B68EE;">[at_least two affected female first-degree relatives]c_22 </span>, because such <span style="white-space:nowrap;color:#D2B48C;">[cases]c_13 </span>are more likely to carry <span style="white-space:nowrap;color:#826E43;">[susceptibility]c_25 </span>alleles . </nobr><p> [ c_2 / c_4,c_0,c_24,c_27 ] <span style="white-space:nowrap;color:#556B2F;">we </span> ////<br/><nobr>analysed <span style="white-space:nowrap;color:#000080;">[227,874 SNPs]c_4 </span>( <span style="white-space:nowrap;color:#708090;">[85_%]c_0 </span>) with <span style="white-space:nowrap;color:#5F9EA0;">[genotypes]c_24 </span>on at_least 80_% of <span style="white-space:nowrap;color:#00CED1;">[the subjects]c_27 </span></nobr><p> [ c_3 / c_3,c_18,c_29 ] <span style="white-space:nowrap;color:#FF3300;">these SNPs </span> ////<br/><nobr>are correlated with 55_% of common <span style="white-space:nowrap;color:#FF3300;">[SNPs]c_3 </span>in the HapMap_CEPH / CEU ( Utah residents with ancestry from <span style="white-space:nowrap;color:#008000;">[northern and western Europe]c_18 </span>) samples at r2 0.8 , and 77_% at r2 0.5 ( mean r2=0.75 ; see <span style="white-space:nowrap;color:#00008B;">[Supplementary_Fig]c_29 </span>. 1 ) ( http://www.hapmap.org/ ) </nobr><p> [ c_8 / c_2 ] <span style="white-space:nowrap;color:#6A5ACD;">coverage </span> ////<br/><nobr>was strongly related to m . a . f . : 70_% of SNPs with m . a . f . 10_% were tagged at <span style="white-space:nowrap;color:#556B2F;">[r2]c_2 </span>0.8 , compared with 23_% of SNPs with m . a . f . 5 – 10_% . </nobr><p> [ c_9 / c_4,c_19,c_2 ] <span style="white-space:nowrap;color:#663399;">The main analyses </span> ////<br/><nobr>were restricted to <span style="white-space:nowrap;color:#000080;">[205,586 SNPs]c_4 </span>that had <span style="white-space:nowrap;color:#AF83BE;">[a]c_19 </span>call rate of 90_% and whose genotype distributions did not differ from Hardy – Weinberg equilibrium in controls ( at <span style="white-space:nowrap;color:#556B2F;">[P]c_2 </span>10 − 5 ) . </nobr><p> [ c_2 / c_3,c_16 ] <span style="white-space:nowrap;color:#556B2F;">we </span> ////<br/><nobr>selected 12,711 <span style="white-space:nowrap;color:#FF3300;">[SNPs]c_3 </span>, approximately 5_% of those typed in <span style="white-space:nowrap;color:#FF436E;">[stage]c_16 </span>1 , . </nobr><p> [ c_3 / c_0,c_13,c_15 ] <span style="white-space:nowrap;color:#FF3300;">These SNPs </span> ////<br/><nobr>were then genotyped in a further 3,990 invasive <span style="white-space:nowrap;color:#708090;">[breast cancer]c_0 </span><span style="white-space:nowrap;color:#D2B48C;">[cases]c_13 </span>and <span style="white-space:nowrap;color:#32CD32;">[3,916 controls]c_15 </span>from the SEARCH study , using a custom-designed oligonucleotide array . </nobr><p> [ c_2 / c_4,c_30,c_31 ] <span style="white-space:nowrap;color:#556B2F;">we </span> ////<br/><nobr>considered <span style="white-space:nowrap;color:#000080;">[10,405 SNPs]c_4 </span>with <span style="white-space:nowrap;color:#438281;">[call rate]c_30 </span>of <span style="white-space:nowrap;color:#4169E1;">[95_%]c_31 </span>that did not deviate from Hardy – Weinberg equilibrium in controls . </nobr><p> [ c_10 / c_17,c_11,c_3 ] <span style="white-space:nowrap;color:#BDB76B;">Comparison of the observed and expected distribution of test statistics </span> ////<br/><nobr>showed some <span style="white-space:nowrap;color:#0000CD;">[evidence]c_17 </span>for <span style="white-space:nowrap;color:#FFD700;">[an inflation]c_11 </span>of the test statistics in both stage 1 ( inflation factor λ=1.03 , 95_% confidence interval ( CI ) 1.02 – 1.04 ) and stage 2 ( λ=1.06 , 95_% CI 1.04 – 1.12 ) , based on the 90_% least significant <span style="white-space:nowrap;color:#FF3300;">[SNPs]c_3 </span>( Fig . 1 ) . </nobr><p> [ c_11 / c_32,c_27,c_13 ] <span style="white-space:nowrap;color:#FFD700;">Possible explanations for this inflation </span> ////<br/><nobr>include <span style="white-space:nowrap;color:#FF1493;">[population stratification]c_32 </span>, cryptic relatedness among <span style="white-space:nowrap;color:#00CED1;">[subjects]c_27 </span>, and differential genotype calling between <span style="white-space:nowrap;color:#D2B48C;">[cases]c_13 </span>and controls . </nobr><p> [ c_2 / c_17,c_26,c_30,c_2,c_19,c_29 ] <span style="white-space:nowrap;color:#556B2F;">There </span> ////<br/><nobr>was <span style="white-space:nowrap;color:#0000CD;">[evidence]c_17 </span>for <span style="white-space:nowrap;color:#6495ED;">[an excess]c_26 </span>of low <span style="white-space:nowrap;color:#438281;">[call rate]c_30 </span>SNPs among the most significant SNPs ( <span style="white-space:nowrap;color:#556B2F;">[P]c_2 </span>0.01 ) in stage 1 , but not in stage 2 , suggesting that some of this effect is <span style="white-space:nowrap;color:#AF83BE;">[a]c_19 </span>genotyping artefact ( <span style="white-space:nowrap;color:#00008B;">[Supplementary_Table]c_29 </span>1 ) . </nobr><p> [ c_11 / c_30,c_13,c_32 ] <span style="white-space:nowrap;color:#FFD700;">the inflation </span> ////<br/><nobr>was present with <span style="white-space:nowrap;color:#438281;">[call rate]c_30 </span>99_% in both <span style="white-space:nowrap;color:#D2B48C;">[cases]c_13 </span>and controls , possibly reflecting <span style="white-space:nowrap;color:#FF1493;">[population sub-structure]c_32 </span>. </nobr><p> [ c_2 / c_20,c_19,c_24 ] <span style="white-space:nowrap;color:#556B2F;">We </span> ////<br/><nobr>computed 1 degree of freedom ( d . <span style="white-space:nowrap;color:#6B8E23;">[f]c_20 </span>. ) <span style="white-space:nowrap;color:#AF83BE;">[association]c_19 </span>tests for each SNP , combining <span style="white-space:nowrap;color:#5F9EA0;">[stages]c_24 </span>1 and 2 . </nobr><p> [ c_2 / c_2,c_29 ] <span style="white-space:nowrap;color:#556B2F;">we </span> ////<br/><nobr>observed more associations than would have been expected by_chance at <span style="white-space:nowrap;color:#556B2F;">[P]c_2 </span>0.05 <span style="white-space:nowrap;color:#00008B;">[( Table]c_29 </span>1 ) . </nobr><p> [ c_4 / c_2,c_5 ] <span style="white-space:nowrap;color:#000080;">One SNP ( dbSNP rs2981582 ) </span> ////<br/><nobr>was significant at the <span style="white-space:nowrap;color:#556B2F;">[P]c_2 </span>10 − 7 level that has been proposed as appropriate for <span style="white-space:nowrap;color:#2F4F4F;">[genome-wide studies]c_5 </span>. </nobr>	</body>
</html>